sdk/python/feast/infra/passthrough_provider.py

```diff
@@ -2,33 +2,34 @@
 from typing import Any, Callable, Dict, List, Optional, Sequence, Tuple, Union
 
 import pandas as pd
-import pyarrow
 import pyarrow as pa
 from tqdm import tqdm
 
-from feast import FeatureService
+from feast.batch_feature_view import BatchFeatureView
 from feast.entity import Entity
 from feast.feature_logging import FeatureServiceLoggingSource
+from feast.feature_service import FeatureService
 from feast.feature_view import FeatureView
+from feast.infra.materialization import BatchMaterializationEngine, MaterializationTask
 from feast.infra.offline_stores.offline_store import RetrievalJob
 from feast.infra.offline_stores.offline_utils import get_offline_store_from_config
 from feast.infra.online_stores.helpers import get_online_store_from_config
-from feast.infra.provider import (
-    Provider,
-    _convert_arrow_to_proto,
-    _get_column_names,
-    _run_field_mapping,
-)
+from feast.infra.provider import Provider, _convert_arrow_to_proto, _run_field_mapping
 from feast.protos.feast.types.EntityKey_pb2 import EntityKey as EntityKeyProto
 from feast.protos.feast.types.Value_pb2 import Value as ValueProto
 from feast.registry import BaseRegistry
 from feast.repo_config import RepoConfig
 from feast.saved_dataset import SavedDataset
+from feast.stream_feature_view import StreamFeatureView
 from feast.usage import RatioSampler, log_exceptions_and_usage, set_usage_attribute
 from feast.utils import make_tzaware
 
 DEFAULT_BATCH_SIZE = 10_000
 
+BATCH_ENGINE_CLASS_FOR_TYPE = {
```

Comment:
this map already exists in `repo_config.py`

Reply:
Yeah I'll remove the dupe there